Cannot build with latest version of gonum and Go modules

Steps to reproduce:

```shell
rm -f go.mod && rm -f go.sum && rm -rf vendor
go mod init `basename $(pwd)` && go mod tidy; go mod vendor
go build -mod=vendor -o ./bin/foo ./cmd/service
```

Error received from build:

```go
# gorgonia.org/gorgonia
vendor/gorgonia.org/gorgonia/graph.go:569:2: cannot use e (type edge) as type graph.Edge in return argument:
	edge does not implement graph.Edge (missing ReversedEdge method)
vendor/gorgonia.org/gorgonia/node.go:756:15: cannot use e (type edge) as type graph.Edge in argument to n.g.SetEdge:
	edge does not implement graph.Edge (missing ReversedEdge method)
```

--- a/go.mod
+++ b/go.mod
@@ -21,7 +21,6 @@ require (
 	gopkg.in/cheggaaa/pb.v1 v1.0.27
 	gorgonia.org/cu v0.9.0-beta
 	gorgonia.org/dawson v1.1.0
-	gorgonia.org/gorgonia v0.9.1
 	gorgonia.org/tensor v0.9.0-beta
 	gorgonia.org/vecf32 v0.7.0
 	gorgonia.org/vecf64 v0.7.0

